watertap/costing/watertap_costing_package.py

```diff
@@ -184,6 +184,16 @@ def build_global_params(self):
         )
         self.add_defined_flow("magnesium chloride", self.magnesium_chloride_cost)
```

Comment:
```suggestion
        self.add_defined_flow("magnesium chloride", self.magnesium_chloride_cost)
```
On second thought, shouldn't this go under the electronp costing and not in the global param block for WT costing?

Reply:
This is the chemical addition registered flow for operating cost, it is similar to electricity and based on convention, it is added in the global param block. For the unit block, we call it by `cost_MgCl2_flow`

Reply:
I moved it to unit model for now